doc/man3/RAND_bytes.pod

```diff
@@ -24,9 +24,10 @@ into B<buf>. An error occurs if the CSPRNG has not been seeded with
 enough randomness to ensure an unpredictable byte sequence.
 
 RAND_priv_bytes() has the same semantics as RAND_bytes().  It is intended to
-be used for generating long-term private keys. If using the default
-RAND_METHOD, this function uses a separate instance of the PRNG so that
-a compromise of the global generator will not affect such key generation.
+be used for generating values that should remain private. If using the
+default RAND_METHOD, this function uses a separate instance of the PRNG
+so that a compromise of the global generator will not affect the secrecy
+of such values.
```

Comment:
Is it really the "global generator" anymore?  Perhaps "a compromise of the PRNG used for public-facing values will not affect the secrecy of values from the private generator"?

Reply:
@kaduk fixed in 7bd4945
@kroeckx does it sound good to you?

Reply:
How about the following formulation?

```
... this function uses a separate "private" PRNG instance so that a 
compromise of the "public" PRNG instance will not affect the 
secrecy of these private values.
```

Reply:
See man [RAND_DRBG(7)](https://github.com/openssl/openssl/blob/master/doc/man7/RAND_DRBG.pod#the-overall-picture) for a nice ascii art picture :-).

Reply:
The image looks a bit distorted in my browser, but not on the console.

Reply:
That's very nice!! I should check Section 7 more often, there is a lot of cool stuff there!